cmd/cloud/setup.go

```diff
@@ -79,6 +98,22 @@ func Setup(cmd *cobra.Command, args []string, client astro.Client, coreClient as
 		return nil
 	}
 
+	// if deployment inspect, create, or udpate commands are used
+	deploymentCmds := []string{"inspect", "create", "update"}
+	if util.Contains(deploymentCmds, cmd.CalledAs()) && cmd.Parent().Use == deploymentCmd {
+		isDeploymentCmd = true
+	}
+
+	// Check for APITokens before API keys or refresh tokens
+	apiToken, err := checkAPIToken(isDeploymentCmd, args)
+	if err != nil {
+		fmt.Println(err)
+		fmt.Println("\nThere was an error using API tokens, using regular auth instead")
```

Comment:
Considering this is going to be used mostly in ci/cd pipelines where regular auth doesnt exist, should we terminate the flow here?

Reply:
yes I'll do that